shiny/ui/_html_deps_py_shiny.py

```diff
@@ -52,3 +53,24 @@ def spin_dependency() -> HTMLDependency:
         source={"package": "shiny", "subdir": "www/shared/py-shiny/spin"},
         stylesheet={"href": "spin.css"},
     )
+
+
+def busy_indicators_dependency() -> HTMLDependency:
+    return HTMLDependency(
+        "shiny-busy-indicators",
+        __version__,
+        source={"package": "shiny", "subdir": "www/shared/py-shiny/busy-indicators"},
+        stylesheet={"href": "busy-indicators.css"},
+        head=TagList(
+            # Default to spinners mode
+            busy_indicators.mode("spinners"),
```

Comment:
If we merge this into this release, we'll need:

```suggestion
            # TODO: uncomment to default to spinners mode
            # busy_indicators.mode("spinners"),
```